extensions/sql-migration/src/api/utils.ts

```diff
@@ -676,6 +693,10 @@ export function getAzureResourceDropdownValues(
 }
 
 export function getResourceDropdownValues(resources: { id: string, name: string }[], resourceNotFoundMessage: string): CategoryValue[] {
+	if (!resources || !resources.length) {
```

Comment:
this seems redundant, isn't this the same as what line 701 is doing?

Reply:
I was running into null exceptions, which is why i added this

Reply:
here is the bug carlos found related to this: https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2123557